AdyenDropIn/Presentation/WrapperViewController.swift

```diff
@@ -8,9 +8,9 @@ import Adyen
 import UIKit
 
 /// :nodoc:
+/// Wrapps ModalViewController and manage it conten's updates
```

Comment:
Wraps ModalViewController and manages it's content updates

Reply:
It is actually some leftovers